src/pages/api/tutorial-feedback.ts

```diff
@@ -132,17 +135,25 @@ const submitFeedback = async (
 		if (updatedRow) {
 			await updatedRow.save()
 		} else {
-			//  eslint-disable-next-line @typescript-eslint/ban-ts-comment
-			// @ts-ignore
-			await sheet.addRow(newRow)
+			await sheet.addRow({ ...newRow })
 		}
 
 		res.status(204).end()
 	} catch (error) {
-		console.error('Unexpected error')
-		console.error(error)
-		res.status(error.status || 500).json({
-			body: { error: 'An unexpected error occurred.' },
+		console.error('Error occurred.')
+
+		let errorMessage = 'An unexpected error occurred.'
+
+		if (error.response?.status === 404) {
+			errorMessage = 'Requested entity was not found.'
+		}
+
+		if (error.response?.status === 400) {
+			errorMessage = 'Invalid grant: account not found.'
+		}
```

Comment:
Tested these errors and these messages were returned in the actual error

Reply:
💭 If they are already coming back in the error.message, do we need to specify them here? Or perhaps we just concat with the original message `An unexpected error  occurred: ${error.message}`

Reply:
I like the idea but the `error.message` for 404 comes back as `'error:0909006C:PEM routines:get_name:no start line' }`. I didn't think that was very clear but if we think otherwise then 👍 . Or maybe there is another common field in the error object that we can use?

Reply:
I suppose that _is_ the actual error message associdate tho. So I'm fine with this!

Reply:
I'll leave it to you to decide 👍